apps/ledger-live-desktop/src/internal/commands/ftsLoadImage.ts

```diff
@@ -0,0 +1,8 @@
+import { Observable, from } from "rxjs";
+import installLanguage, {
+  LoadImageEvent,
+  LoadImageRequest,
+} from "@ledgerhq/live-common/hw/ftsLoadImage";
+const cmd = (input: LoadImageRequest): Observable<LoadImageEvent> => from(installLanguage(input));
```

Comment:
🤔 are you sure it's `installLanguage` ? i think the code is correct, just you misnamed the default import ;)

Reply:
Ah lol bad copy paste, thanks 😆